packages/codebytes/src/codeByteEditor.tsx

```diff
@@ -12,38 +12,29 @@ import { LanguageSelection } from './languageSelection';
 import { trackUserImpression } from './libs/eventTracking';
 import { CodeByteEditorProps } from './types';
 
-const editorStates = states({
-  isIFrame: { height: '100vh' },
-});
-
 const editorBaseStyles = system.css({
   border: 1,
   borderColor: 'gray-900',
   display: 'flex',
   flexDirection: 'column',
-  height: '25rem',
-  width: '43rem',
   overflow: 'hidden',
+  minHeight: '25rem',
```

Comment:
for some reason we need this for the first styleguide story to work, i'm not totally sure why the height prop works for the second but not the first, but this solves it 🤷‍♀️

Reply:
A thought: we only needed `system.css` to work with the single `isIFrame` prop. If we're moving away from passing in the prop, and passing in styles directly, we can just do:

const EditorContainer = styled(Background)({
  border: 1,
  borderColor: 'gray-900',
  display: 'flex',
  flexDirection: 'column',
  minHeight: '25rem',
  maxWidth: '43rem',
});

And we can remove the `system` and `StyledProps` dependencies from the import. Early, there was no way of getting `isIFrame` to work otherwise